addons/account_edi/models/account_move.py

```diff
@@ -268,6 +275,14 @@ def _post(self, soft=True):
             self.env.ref('account_edi.ir_cron_edi_network')._trigger()
         return posted
 
+    def force_cancel_edi(self):
```

Comment:
```suggestion
    def button_force_cancel(self):
```
prefixing with 'button' makes it clear that it is a button. (it's already the case for all the other buttons.)